test/Libraries/DynamoPythonTests/IronPythonTests.csproj

```diff
@@ -117,6 +119,10 @@
       <Name>PythonNodeModels</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\src\Tools\DynamoShapeManager\DynamoShapeManager.csproj">
+      <Project>{263fa9c1-f81e-4a8e-95e0-8cdae20f177b}</Project>
+      <Name>DynamoShapeManager</Name>
```

Comment:
just curious about the requirement for these dependencies.

Reply:
So, when you inherit from `DynamoModelTestBase` and want to customize `CreateStartConfiguration` you pass some parameters that will complain because you don't have these.

Reply:
ahh, okay, generally I would say avoid loading libraries that we don't need during the test as these types will be imported into the VM over and over again for each test which can be kind of slow. So if we write 1000 ironPython tests we'll load all the geometry types into the VM foreach test.